sympy/utilities/lambdify.py

```diff
@@ -588,6 +616,276 @@ def flat_indexes(iterable):
     expr = lambdarepr(expr)
     return "lambda %s: (%s)" % (args, expr)
 
+if PY3:
+    def _is_safe_ident(ident):
+        from keyword import iskeyword
+
+        return isinstance(ident, str) and ident.isidentifier() \
+                and not iskeyword(ident)
+else:
+    _safe_ident_re = re.compile('^[a-zA-Z_][a-zA-Z0-9_]*$')
+
+    def _is_safe_ident(ident):
+        from keyword import iskeyword
+
+        return isinstance(ident, str) and _safe_ident_re.match(ident) \
+            and not (iskeyword(ident) or ident == 'None')
+
+def _generate_unpack_iterable(lvalues, rvalue):
+    """Generate argument unpacking code for _lambdastrimpl.
+
+    This method is used when the input value is iterable.
+    """
+    def unpack_lhs(lvalues):
+        return '[{}]'.format(', '.join(
+            unpack_lhs(val) if iterable(val) else val for val in lvalues))
+
+    return ['{} = {}'.format(unpack_lhs(lvalues), rvalue)]
+
+def _generate_unpack_indexable(lvalues, rvalue):
+    """Generate argument unpacking code for _lambdastrimpl.
+
+    This method is used when the input value is not interable,
+    but can be indexed.
+    """
+    from sympy import flatten
+
+    def flat_indexes(elems):
+        n = 0
+
+        for el in elems:
+            if iterable(el):
+                for ndeep in flat_indexes(el):
+                    yield (n,) + ndeep
+            else:
+                yield (n,)
+
+            n += 1
+
+    indexed = ', '.join('{}[{}]'.format(rvalue, ']['.join(map(str, ind)))
+                            for ind in flat_indexes(lvalues))
+
+    return ['[{}] = [{}]'.format(', '.join(flatten(lvalues)), indexed)]
+
+def _generate_numpy_arg_wrapping(args):
+    """Generate numpy argument wrapping code for _lambdastrimpl."""
+    integer_names = ', '.join(cls.__name__ for cls in integer_types)
+
+    lines = ['builtin_numerics = ({}, float, complex)'.format(integer_names)]
```

Comment:
It would be better to just add this to the function namespace. Otherwise it has to be run every time the function is run.

Reply:
Actually, all this code was meant to be deleted!  It's now inside the _EvaluatorPrinter class and sub-classes.  But yes, this can be done in the _NumpyEvaluatorPrinter class definition, instead of every function call.  Is it best if I make a new pull request to change this and remove the unused code?

Reply:
This was merged so yes, you need to create a new pull request.